include/vara/Feature/FeatureModel.h

```diff
@@ -357,9 +323,14 @@ template <> struct GraphWriter<vara::feature::FeatureModel *> {
       for (auto *Child : *Node) {
         emitCluster(Child, Indent + 2);
         O.indent(Indent + 2);
-        emitEdge(Node, Child,
-                 llvm::formatv("arrowhead={0}",
-                               Child->isOptional() ? "odot" : "dot"));
+        auto *F = llvm::dyn_cast<vara::feature::Feature>(Child);
+        if (F) {
```

Comment:
```suggestion
        
        if (auto *F = llvm::dyn_cast<vara::feature::Feature>(Child); F) {
```
If you only use the value inside the if, you can use a if initializer. Makes for better scoping. `F` will be out of scope after the `if`, making it impossible to be used after wards, which would be good in the case `F == nullptr`